react/Footer.js

```diff
@@ -1,16 +1,10 @@
 import React, { Suspense } from 'react'
 import { ExtensionPoint, useChildBlock, NoSSR } from 'vtex.render-runtime'
 import { useDevice } from 'vtex.device-detector'
-import useFold from './hooks/useFold'
 
 const LegacyFooter = React.lazy(() => import('./legacy/Footer'))
 
 const Footer = props => {
-  /** This is a temporary fix--this behaviour is supposed
```

Comment:
truly temporary 👏

Reply:
first ever temporary fix to be actually temporary